src/collection/collection.service.ts

```diff
@@ -1279,7 +1279,7 @@ export class CollectionService {
 		order: Record<string, 'asc' | 'desc'> | Record<string, 'asc' | 'desc'>[],
 		contentTypeId: ContentTypeNumber.collection | ContentTypeNumber.bundle,
 		filterString: string | undefined
-	): Promise<Avo.Collection.Collection[]> {
+	): Promise<Collection[]> {
```

Comment:
OPTIONAL: it might be better to add the share_type?: ... to the Avo.Collection.Collection type, so we can keep using a single type?
only if this isn't too much work to change.

Reply:
I tried to keep it consistent/aligned with assignments